Doc/howto/decorators.rst

```diff
@@ -190,6 +222,16 @@ If you do not use :meth:`functools.update_wrapper` when implementing a decorator
        decorated.__wrapped__ = obj
        return decorated
 
+.. It's worth really pushing use of ``functools.wraps`` and/or
+   ``functools.update_wrapper`` here.  Doing that will provide better
+   forward compatibility if additional attributes are added that should
+   be propagated to the wrapper.
+
+   The ``__wrapped__`` attribute is really for introspection support,
```

Comment:
There are only two reasons I've been able to think of that make ``__wrapped__`` useful:

1. introspection APIs that are used to determine how things were actually assembled, and

2. debugging support.

I know many programmers like having an exposed reference to what was "originally there", but most are never used, or are only used for debugging when whatever they provide was originally being developed.  I'm wary of "preserving a reference" as a use case or goal in itself.  It can be a perfectly fine way to provide or support some other goal, though.

We should be clear that the goal isn't exposing a reference, but supporting some other activity that uses introspection.